src/main/CommandLine.cpp

```diff
@@ -419,23 +371,21 @@ clara::Opt
 inMemoryParser(bool& inMemory)
 {
     return clara::Opt{inMemory}["--in-memory"](
-        "(DEPRECATED) store working ledger in memory rather than database");
+        "(DEPRECATED) flag is ignored and will be removed soon.");
```

Comment:
This was specifically a request from platform. there are some Horizon configs that use this flag that are niche and no longer supported, but still exist in the codebase. They wanted us to just ignore the flags for now and not crash so that we don't have a breaking point release.